src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -479,7 +479,8 @@ public void Evaluate(IParameterSet parameters, IVariableCollection rootVariableC
 
         internal static SimpleConfigModel FromJObject(IEngineEnvironmentSettings environmentSettings, JObject source, ISimpleConfigModifiers configModifiers = null, JObject localeSource = null)
         {
-            ILocalizationModel localizationModel = LocalizationFromJObject(localeSource);
+            ILogger logger = environmentSettings.Host.Logger;
+            _ = LocalizationModelDeserializer.TryDeserialize(localeSource, logger, out ILocalizationModel localizationModel);
```

Comment:
suggestion for consistency: let's split this method up:
- first it loads basic template.json
- then `LocalizationModelDeserializer.TryDeserialize(localeSource, logger, out ILocalizationModel localizationModel)`
- then `LocalizationModelDeserializer.VerifyLocalizationModel(localizationModel, config, logger)`
- if successful new method `LoadLocalization` where properties of SimpleConfigModel will be overwritten from `localizationModel`.

This should not be too difficult to split.